iec61131-st/bit_num_conversion.st

```diff
@@ -0,0 +1,923 @@
+(********************
+*
+* Converts LWORD to LREAL
+*
+*********************)
+FUNCTION LWORD_TO_LREAL : LREAL
+VAR_INPUT
+	in : LWORD;
+END_VAR
+	LWORD_TO_LREAL := in;
```

Comment:
WORD to REAL functions should be binary transfers, what this code does is convert the values
You should use a pointer transfer here.